src/components/Composer.vue

```diff
@@ -15,82 +16,165 @@
 				:placeholder="t('mail', 'Select account')"
 				:clear-on-select="false"
 				@select="onAliasChange" />
+				<button 
+					title="" 
+					:class="['icon-more',{'active':!autoLimit}]"
+					@click.prevent="autoLimit = !autoLimit">
+				</button>
```

Comment:
this whole part doesnt work for me and im not sure what is supposed to do, i click it and nothing happens

Reply:
Oh.. i'll check and fix this. Stranger things...

Reply:
![Peek 2022-05-06 16-44](https://user-images.githubusercontent.com/3595562/167145240-e5598c0b-24e4-4f5a-ac80-8f4b09306a97.gif)
https://github.com/sazanof/nextcloud_mail/commit/f924645383eaed44befbb995e8e1e4eb7f7a3c4e
@GretaD 

_**Measure it seven times, cut it off once. In my case, you can rephrase: seven commits, one pull request =).**_ 

I think it's all because of my inattention when correcting conflicts, and as a result - because of this comma. I've corrected it now. Launched rpm run watch (and build tool) I cleaned the cache, and checked (not like that time =))
GIF in the attachment. Thank you!

Reply:
haha :) perfect, thanks a lot. So, i didnt know what to expect from the 3 dot action menu, so "not working" was not a good explanation or  clear from my side, sorry about that. Yes, i can confirm the gif now :)

Reply:
I was confused by the three dot menu. Would it be possible to move its functionality to the `+1` badge? E.g. if I click on `+1` it shows all recipients in multiple lines.

Reply:
> not a good explanation or clear from my side

![!Peek 2022-05-11 11-18](https://user-images.githubusercontent.com/3595562/167804028-5f3d328b-1d2d-49c7-a38a-c4751e292af4.gif)
And what if you do it like this: 

Has it become less unclear? =) 
In theory, you can hang events on the counter (+1, +5....) but, I'm not sure that's a good idea.
You can immediately show the entire list of recipients, but on fhd 15.6" laptops, the pop-up window seems small and the recipients have the format (John Doedoeodeo (doedoeodeodeo@jjj.com )) they are placed in a row only one piece at a time, like most postal addresses. What do you think? If it's OK, I'll make a commit with changes (There is still something to make).
PS also found a bug with the position of the loading indicator when searching for contacts, corrected and updated the icons on materialicons.

Reply:
@GretaD I told you, I'm going to ask stupid questions? In short, I don't know how to work with git yet and have done nonsense to the campaign) How can I fix everything back now?
PR close after force push (Now I know it's bad practice)
1e8d3b9f9b14c17ff7ce3f40e0f3e87c9cf336e0
733e5ff276ef596662dbc310cb262ff2a78255dd
All commits are lost =( 
Can I open a new PR or what is better to do?
___
Good practice - having practice